datapackage/resource_new.py

```diff
@@ -0,0 +1,267 @@
+from __future__ import absolute_import
```

Comment:
@amercader 
Yes of course:
> To avoid overwhelming diff in this PR I've only added new Resource class based on tableschema.Table with tests. In the second PR I'll integrate it with DataPackage class.

So no plans to have both new/legacy resource but just to simplify changes reviewing. If you and @akariv prefer all in this PR I could continue to work in this PR.